app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -4,7 +4,9 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Customer(var name: String, var phone: String, @SerializedName("id") var customerId:Int = 0 ) {
+data class Customer(var name: String,
+                    var phone: String,
+                    @SerializedName("id") var customerId:Int = 0 ) {
```

Comment:
Added some line breaks in class parameters for readability per kotlin best practices